client_test.go

```diff
@@ -59,6 +59,8 @@ const UpdateFeatureStateResponseJson = `
   "change_request": null
 }
 `
+const FeatureID int64 = 1
```

Comment:
Note: they are hardcode here because go does not support string interpolation for constants

Reply:
Should we add the projectUUID here too?

Reply:
Isn't more relevant here: https://github.com/Flagsmith/flagsmith-go-api-client/pull/3/files#diff-4dc56e64f310cd8c12bb00f75c09e83f5d875563ea9901788e525487dc6788b3R175?

Reply:
I'm confused why we are still hardcoding it in so many places though?

Reply:
well,  because go does not support string interpolation for constants